src/razeedash/Sender.js

```diff
@@ -46,37 +47,46 @@ module.exports = class RazeedashSender {
   }
 
 
-  reset(){
-    this.flush();
-    this._sentSelflinks={};
+  reset() {
```

Comment:
How do we avoid losing the objects in the outbound array when flush is called?

Reply:
im not sure i understand the question. We call flush (which send all resources in the outbound array and puts the send promise in `this._dsa.getSendPromises`), then we wait for the `this._dsa.getSendPromises` to finish here https://github.com/razee-io/Watch-keeper/pull/38/files#diff-02952b85e05587f619c17cc67efa16deR61